### PR TITLE
Fixed a ton of typos

### PR DESCRIPTION
**Pull request by JrodManU**
Read through all the docs and corrected some typos for you guys :)

### CODE REVIEW

**Comment by rdegges** [comment]
This is awesome, thank you very much =)

File: docs/changelog.rst [hunk 1]
```diff
@@ -141,7 +141,7 @@ Version 2.0.3
 **Released on September 8, 2015.**
 
 - Fixing bug in the ``groupsRequired`` authorization middleware -- it was using
-  a deprecated option, which was causing the library to throw an error if a user
+  a depreciated option, which was causing the library to throw an error if a user
```
Comment (erbridge) [inline docs/changelog.rst]: This was right before. https://en.wikipedia.org/wiki/Deprecation

**Comment by JrodManU** [comment]
Huh, I have always read that word as depreciation. Learn new things everyday! Will fix

**Comment by JrodManU** [comment]
Submitted a new pull request with it fixed. Good thing you caught that @erbridge

**Pull request merged**